lib/stat_tracker.rb

```diff
@@ -1,7 +1,7 @@
 require 'csv'
-require_relative 'game_stats'
-require_relative 'league_stats'
-require_relative 'team_stats'
+require_relative '../modulars/game_stats'
+require_relative '../modulars/league_stats'
```

Comment:
what are these?

Reply:
```suggestion
require_relative '../modulars/league_stats'
```So Ian mentioned during our review that we should put the modules in a separate folder for organization. So I did but then they were no longer located in the lib file with the rest of them so I have to update the require_relative so that they pulled from the new directory that is now their home.